cmd/notary/main_test.go

```diff
@@ -103,6 +103,7 @@ var exampleValidCommands = []string{
 	"status repo",
 	"publish repo",
 	"add repo v1 somefile",
+	"addhash repo targetv1 --sha256 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 10",
```

Comment:
I'll look into what distribution does -- however, I'm not sure if we want this flexibility because a user typo (ex: `sha255:..`) could ruin the target entirely, and it wouldn't be noticeable until verifying after a publish.  It seems like a more usable flow to be strict up front and ensure that hashes are correctly inputted?